styles.css

```diff
@@ -1,504 +1,1051 @@
+    border: 2px solid white;
+    border-radius: 20px;
+    margin: 100px;
+    background-color: #fff;
+  }
+
+  .Aa {
+    order: 2;
+  }
+
+  .Bb {
+    order: 1;
+  }
+
+  .block1 {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+
+  .card-img {
+    padding: 30px;
+    width: 100%;
+  }
+
+  .block2 {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-left: 60px;
+  }
+
+  .box1 {
+    display: flex;
+    flex-direction: column;
+    margin: 40px -1px;
+    position: relative;
+    bottom: 27px;
+  }
+
+  .title1 {
+    width: 100%;
+    height: 46px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    color: #091e42;
+  }
+
+  .frame1 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 1px 40px;
+    height: 32px;
+    margin-top: -1px;
+    margin-left: -55px;
+  }
+
+  .A {
+    margin-right: -39px;
+  }
+
+  .Li {
+    width: 107px;
+    height: 16px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    padding-left: 18px;
+    list-style: none;
+    color: #7a869a;
+  }
+
+  .box2 {
+    width: 100%;
+    height: 76px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    margin-bottom: 150px;
+    position: relative;
+    bottom: 80px;
+  }
+
+  .card-para {
+    color: #344563;
+    max-width: 90%;
+    min-width: 20%;
+    height: 65px;
+  }
+
+  .box2-ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1px;
+    gap: 8px;
+    width: 214px;
+    height: 24px;
+    list-style: none;
+  }
+
+  .box2-li {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    background: #ebebff;
+    border-radius: 8px;
+    border: none;
+    padding: 5px;
+  }
+
+  .big-btn {
+    width: 126px;
+    height: 48px;
+    margin-top: 15px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    color: #396df2;
+  }
+
+  .icon {
+    color: #c1c7d0;
+    font-size: 8px;
+    position: relative;
+    bottom: 1px;
+    right: 1px;
+  }
+
+  /* .section {
+    margin-top: 50px;
+  } */
+
+  /* about page  */
+
+  .desktop-v {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 930px;
+    border-top-right-radius: 100px;
+  }
+
+  .section1 {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    height: 1000px;
+    background-color: #fff;
+    padding: 30px;
+    padding-right: 1px;
+  }
+
```

Comment:
- [ ] Your desktop version of the About Me section is fantastic. However, to ensure consistency with the Figma template, there are some issues that need to be addressed, as listed below:
- The left block of the page should take up 40% of the width, while the right block should occupy the remaining 60%.
- The social links and the "Get my resume" button should be aligned with the other elements.
- The "JavaScript," "CSS," and "HTML" languages should be aligned as per the template requirements. Please also check the font family of each language.
- As mentioned earlier, kindly check the button background to match the Figma button design.

**Your design**

![Screenshot from 2023-03-31 18-35-05](https://user-images.githubusercontent.com/105117832/229250588-cbd44434-572c-457b-9337-4573abc77a5e.png)

**Figma**

![Screenshot from 2023-03-31 18-34-50](https://user-images.githubusercontent.com/105117832/229250606-7ad93406-e20a-4ce8-849e-d05c50efd568.png)